### PR TITLE
Made changes to implement deleteMember, deleteFact, createFact, and updateFact

### PR DESCRIPTION
**Pull request by mhope21**
I started off with a big problem. I knew everything was right in my deleteMember, but it still wasn't working. I went through my controllers, my routes, and the js and html files. I couldn't find anything. I put print statements everywhere, none of them even worked for deleteMember. I wasn't getting errors or any indication that I had clicked the button. Finally, I cleared my browser cache. everything worked. End of a 3 hr headache. 

Also, I was confused about the branch instructions. One spot said lesson14, another said fetch.
So we are on the fetch branch, I hope that is ok.

I decided not to do the cors assignment because I am knee-deep in my final project, and I am using the rack-cors gem in it. I will be studying the resource closely though.

### CODE REVIEW

File: app/views/layouts/mailer.html.erb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+<!DOCTYPE >
```
Comment (dsinn) [inline app/views/layouts/mailer.html.erb:1]: This is a curious change. I wouldn't expect it to break anything, but still.

File: app/controllers/api/v1/members_controller.rb [hunk 2]
```diff
@@ -13,9 +13,12 @@ def index
 
   # GET /members/:id
   def show
+    Rails.logger.debug('I have been called.')
     if check_access
       # your code goes here
+      Rails.logger.debug('Access checked')
       render json: {member: @member}
+      Rails.logger.debug('member shown')
```
Comment (dsinn) [inline app/controllers/api/v1/members_controller.rb:21]: Typically you wouldn't check in all this debug code.

**Review by dsinn — APPROVED** [review]
> I started off with a big problem. I knew everything was right in my deleteMember, but it still wasn't working. I went through my controllers, my routes, and the js and html files. I couldn't find anything. I put print statements everywhere, none of them even worked for deleteMember. I wasn't getting errors or any indication that I had clicked the button. Finally, I cleared my browser cache. everything worked. End of a 3 hr headache.

Sorry to hear that. I wonder if [Chrome's network tool](https://developer.chrome.com/docs/devtools/network) (or the equivalent in your browser of choice) would've helped identify this sooner. We don't teach it in Herring, but it's so valuable for debugging at the browser level.

> Also, I was confused about the branch instructions. One spot said lesson14, another said fetch.
> So we are on the fetch branch, I hope that is ok.

Yeah. For me, you could name your branch anything you'd like as long as the Airtable form is filled out correctly, and I'd still be able to review it just fine. I've created Code-the-Dream-School/R7-rest#26 regardless.

**Comment by mhope21** [comment]
Thanks for the tips. I will definitely check this [Chrome's network tool](https://developer.chrome.com/docs/devtools/network) out!

**Pull request merged**